### ISSUE TITLE
youtube-ios-player-helper pod version 0.1.2 can't pause video.

### ISSUE THREAD

**Issue by yukihariguchi** [youtube/youtube-ios-player-helper#69]
youtube-ios-player-helper pod version 0.1.1 can pause video.
But youtube-ios-player-helper pod version 0.1.2 can't pause video.

Please check.
Thanks.

**Comment by thidayatullah** [comment]
How can I get pod version of 0.1.2 ? I still getting version 0.1.1 even after updating it using 'pod update'

**Comment by samiq** [comment]
just clone this repository and start playing from it ;)

**Comment by samiq** [comment]
seems like this was introduced by [this commit](https://github.com/youtube/youtube-ios-player-helper/commit/d8fcf9a5b075ba2bf5e4626bf05434956ad49669#diff-3c387ba9f73c6788962a842d28eed98d) to force replay on low connection, problem is that this approach treat positives as false positives (like when the user tabs on the player to pause the playing) ... since there is no way to discriminate this action from pause due to low bandwidth.

so to fix this 1 of 2 things needs to happen:

- find a way to remove the pause on tap from the video and replace with our own code
- add a property to force play only if wanted, so we the developers decide if we want this behavior or not

**Comment by samiq** [comment]
I have [committed](https://github.com/leaffm/youtube-ios-player-helper/commit/f4e9d7d575866a04c04b8b1da8d3becf90016287) a temporary solution in my fork that bring the pausing back when the user taps on the video (though this solution block the force video playing) as for me I don't need this behavior.

**Comment by Anislav** [comment]
@samiq, you are right about the root of this issue. We are using custom controls and that is why we did not experience this problem.

Even more a lot of commits were merged in the master branch without testing the final result. Some of these commits address the same issues.

I think that @iulukaya should address these issues. From my point of view the master branch is not usable anymore.

**Comment by borut-t** [comment]
This surely works in version 0.1.1.

**Comment by Gazzini** [comment]
@samiq Thanks for your branch; now at least the pause functionality is working for me again! I agree with @Anislav that perhaps this master branch is quite useless unless properly maintained... I spent several hours debugging this because the simplest of delegate methods was not working correctly.

**Comment by jaiminios** [comment]
which state call when tap on done button?

**Comment by RGatin** [comment]
I am using youtube-ios-player-helper library to play youtube videos in my app.
There is a small time delay between you press play button and video starts playing in the full screen. So I want to hide controls in this between time.

How can I do this?
